app-frontend/src/app/core/services/geocode.service.js

```diff
@@ -0,0 +1,34 @@
+/* global $ */
+/* eslint-disable camelcase */
+const appId = 'mXP4DZFBZGyBmuZBKNeo';
+const appCode = 'kBWb6Z7ZLcuQanT_RoP60A';
```

Comment:
That's a good idea since it wouldn't work for anyone deploying from their own infrastructure.